modules/ROOT/pages/dataweave-pattern-matching.adoc

```diff
@@ -97,9 +97,11 @@ output application/json
 }
 ----
 
-For use cases where you need a boolean result based on whether a value
-matches or not, you can use the `matches` function instead of using pattern
-matching.
+For use cases in which the input is a String, you can also use the xref:dw-core-functions-match.adoc[match] _function_ in the `dw::Core` module to test the string against a regex pattern. 
```

Comment:
```suggestion
For use cases in which the input is a String, you can also use the xref:dw-core-functions-match.adoc[match] _function_ in the `dw::Core` module to test the string against a regex pattern (it returns an array with the string that matches the entire regex and its capture groups). 
```
Something like that.
I think that the difference between match and matches functions wasn't clear.

Reply:
Converted the parenthetical into a regular sentence.